packages/teams-js/src/internal/mediaUtil.ts

```diff
@@ -188,12 +188,12 @@ export function validateViewImagesInput(uriList: media.ImageUri[]): boolean {
  *
  * @internal
  */
-export function validateScanBarCodeInput(barCodeConfig: media.BarCodeConfig): boolean {
+export function validateScanBarCodeInput(barCodeConfig?: media.BarCodeConfig): boolean {
```

Comment:
>? [](http://example.com/codeflow?start=54&length=1)

This code was already handling the `undefined` case for this parameter, despite not showing so in the typing. Now it clearly indicates that.